specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingSubscription.json

```diff
@@ -400,244 +749,605 @@
         },
         "invoiceSectionId": {
-          "description": "The ID of the invoice section to which the subscription is billed.",
+          "description": "The fully qualified ID that uniquely identifies an invoice section.",
+          "type": "string",
+          "readOnly": true
+        },
+        "invoiceSectionDisplayName": {
+          "description": "The name of the invoice section.",
+          "type": "string",
+          "readOnly": true
+        },
+        "invoiceSectionName": {
+          "description": "The ID that uniquely identifies an invoice section.",
+          "type": "string",
+          "readOnly": true
+        },
+        "nextBillingCycleDetails": {
+          "description": "Billing cycle details of the product.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/NextBillingCycleDetails"
+            }
+          ]
+        },
+        "offerId": {
+          "description": "The offer ID for the subscription. This field is only available for the Microsoft Online Services Program billing accounts.",
+          "type": "string",
+          "readOnly": true
+        },
+        "productCategory": {
+          "description": "Category of the subscription. Possible values include: ReservationOrder, SavingsPlanOrder, AzureSupport, UsageBased, Hardware, SaaS, SeatBased, and Software.",
+          "enum": [
+            "Other",
+            "ReservationOrder",
+            "SavingsPlanOrder",
+            "AzureSupport",
+            "UsageBased",
+            "Hardware",
+            "SaaS",
+            "SeatBased",
+            "Software"
+          ],
+          "type": "string",
           "readOnly": true,
-          "type": "string"
+          "x-ms-enum": {
+            "name": "BillingSubscriptionCategory",
+            "modelAsString": true
+          }
         },
-        "lastMonthCharges": {
-          "$ref": "./types.json#/definitions/Amount",
-          "description": "The last month charges.",
+        "productType": {
+          "description": "Type of the product for which the subscription is purchased.",
+          "type": "string",
           "readOnly": true
         },
-        "monthToDateCharges": {
-          "$ref": "./types.json#/definitions/Amount",
-          "description": "The current month to date charges.",
+        "productTypeId": {
+          "description": "Id of the product for which the subscription is purchased.",
+          "type": "string",
           "readOnly": true
         },
-        "reseller": {
-          "$ref": "#/definitions/Reseller",
-          "description": "Reseller for this subscription.",
+        "purchaseDate": {
+          "format": "date-time",
+          "description": "Purchase date of the product in UTC time.",
+          "type": "string",
           "readOnly": true
         },
+        "quantity": {
+          "format": "int64",
+          "description": "The number of licenses purchased for the subscription.",
```

Comment:
Not all quantity-based products are licenses. Can we make the description more general?

Reply:
Also, some products have fulfillmentUnits (e.g. Savings Plan, Databricks RI, soon SSC as well). We should consider including that as well.

Reply:
@ramkd12 @amberbhargava to comment here